### PR TITLE
[risk=no] RW-6261 React Tooltip conversion

### PR DESCRIPTION
**Pull request by SrushtiGangireddy**
Converting tooltips to react

### CODE REVIEW

File: public-ui/src/app/data-browser/components/tooltip/tooltip-react.component.tsx [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+  styleUrls: ['./tooltip.component.css', '../../../styles/page.css'],
+  encapsulation: ViewEncapsulation.None,
+})
+export class TooltipWrapperComponent extends BaseReactWrapper {
+  @Input() public label: string;
+  @Input() public searchTerm: string;
+  @Input() public action: string;
+  @Input() public tooltipKey: string;
+  @Input() public shape: string;
+  @Input() public className: string;
```
Comment (dolbeew) [inline public-ui/src/app/data-browser/components/tooltip/tooltip-react.component.tsx]: Do we need these two props? AFAICT every instance of `app-tooltip-react` uses the same values, 'info-standard' and 'is-solid info-icon'. If that's the case, we can just set them directly in the render function.

File: public-ui/src/app/data-browser/views/ehr-view/ehr-view.component.ts [hunk 2]
```diff
@@ -36,6 +36,7 @@ export class EhrViewComponent implements OnChanges, OnInit, OnDestroy {
   prevSearchText = '';
   searchResult: ConceptListResponse;
   items: any[] = [];
+  testReact: boolean;
```
Comment (dolbeew) [inline public-ui/src/app/data-browser/views/ehr-view/ehr-view.component.ts]: This already exists below, line 75. Got an error on dev-up

File: public-ui/src/app/data-browser/views/quick-search/quick-search.component.ts [hunk 3]
```diff
@@ -65,6 +64,7 @@ export class QuickSearchComponent implements OnInit, OnDestroy {
   testReact: boolean;
   showStatement: boolean;
   cope: boolean;
+  testReact: boolean;
```
Comment (dolbeew) [inline public-ui/src/app/data-browser/views/quick-search/quick-search.component.ts:66]: This already exists above, line 64. Got an error on dev-up

File: public-ui/src/app/data-browser/components/tooltip/tooltip-react.component.tsx [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+import * as React from 'react';
+import { FunctionComponent } from 'react';
+import * as ReactDOM from 'react-dom';
+import { BaseReactWrapper } from '../../../data-browser/base-react/base-react.wrapper';
+import { triggerEvent } from '../../../utils/google_analytics';
+import { getTooltip, tooltips } from '../../services/tooltip.service';
+
+const containerElementName = 'root';
+
+const ClrIcon = ({className = '', ...props}) => {
```
Comment (dolbeew) [inline public-ui/src/app/data-browser/components/tooltip/tooltip-react.component.tsx]: Nit: I think this is fine here for now but may want to make it reusable in a utils file or something when we start converting more components with Clarity icons

**Review by dolbeew — APPROVED** [review]
Looks good overall, just a few comments.

File: public-ui/src/app/data-browser/views/ehr-view/ehr-view.component.ts (hunk 2 above)
Comment (dolbeew) [reply public-ui/src/app/data-browser/views/ehr-view/ehr-view.component.ts]: Actually, looks like you fixed this before I submitted my review

File: public-ui/src/app/data-browser/components/tooltip/tooltip-react.component.tsx (hunk 4 above)
Comment (SrushtiGangireddy) [reply public-ui/src/app/data-browser/components/tooltip/tooltip-react.component.tsx]: Thanks agree. Moved it to be re-used.

File: public-ui/src/app/data-browser/components/tooltip/tooltip-react.component.tsx (hunk 1 above)
Comment (SrushtiGangireddy) [reply public-ui/src/app/data-browser/components/tooltip/tooltip-react.component.tsx]: Thanks. Setting values now in the same render.

**Pull request merged**